sdk/storage/storage-blob-changefeed/README.md

```diff
@@ -0,0 +1,170 @@
+# Azure Storage Blob Change Feed client library for JavaScript
+
+> Server Version: 2019-12-12
+
+The purpose of the change feed is to provide transaction logs of all the changes that occur to the blobs and the blob metadata in your storage account. The change feed provides ordered, guaranteed, durable, immutable, read-only log of these changes. Client applications can read these logs at any time. The change feed enables you to build efficient and scalable solutions that process change events that occur in your Blob Storage account at a low cost.
+
+This project provides a client library in JavaScript that makes it easy to consume the change feed.
+
+Use the client libararies in this package to:
+  - Reading change feed events, all or within a time range
+  - Resuming reading events from a saved position
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob-changefeed) |
+[Package (npm)](https://www.npmjs.com/package/@azure/storage-blob-changefeed/) |
+[API Reference Documentation](https://docs.microsoft.com/javascript/api/@azure/storage-blob-changefeed) |
+[Product documentation](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-change-feed) |
+[Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob-changefeed/samples) |
+
+## Getting started
+
+**Prerequisites**: You must have an [Azure subscription](https://azure.microsoft.com/free/) and a [Storage Account](https://docs.microsoft.com/azure/storage/blobs/storage-quickstart-blobs-portal) to use this package. If you are using this package in a Node.js application, then Node.js version 8.0.0 or higher is required.
+
+### Install the package
+
+The preferred way to install the Azure Storage Blob client library for JavaScript is to use the npm package manager. Type the following into a terminal window:
+
+```bash
+npm install @azure/storage-blob-changefeed
+```
+
+### Authenticate the client
+
+This library use a `BlobServiceClient` to initialize, refer to [storage-blob](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob#authenticate-the-client) for how to authenticate a `BlobServiceClient`.
+
+### Compatibility
+
+For this perview, this library is only compatible with Node.js.
+
+### CORS
```

Comment:
nit: since we don't support browsers yet, maybe only add CORS section when the support is added?

Reply:
done